CIP-0129/README.md

```diff
@@ -0,0 +1,162 @@
+
+**Example 1**
+TX ID in Hex - `0000000000000000000000000000000000000000000000000000000000000000`
+Gov Action index in HEX - `11` (number 17)
+gov action ID - `000000000000000000000000000000000000000000000000000000000000000011`
+bech32 - `gov_action1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqpzklpgpf`
+
+**Example 2**
+TX ID in Hex - `1111111111111111111111111111111111111111111111111111111111111111`
+Gov Action index in HEX - `00` (number 0) 
+gov action ID - `111111111111111111111111111111111111111111111111111111111111111100`
+bech32 - `gov_action1zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zygsq6dmejn`
+
+### Bech32 Encoding
+
+| Prefix        | Meaning                                                 | Contents                                                          |
+| ------------- | --------------------------------------------------------| ----------------------------------------------------------------- |
+| `drep`        | DRep identifier                                         | DRep Credential                                                   |
+| `cc_hot`      | CC Hot identifier                                       | CC Hot Credential                                                 |
+| `cc_cold`     | CC Cold identifier                                      | CC Cold Credential                                                |
+| `gov_action`  | gov action identifier                                   | gov action tx id concatenated with index                          |
+
+
+### Identifier Test Vector
+
+We can define a complete identifier as per the spec above by combining the header and the key, see below
+
+#### Constitutional Committee Hot Credential
+
+Key - `00000000000000000000000000000000000000000000000000000000`
+Type - `Key Hash`
+
+Identifier - `0200000000000000000000000000000000000000000000000000000000`
+
+Bech32 - `cc_hot1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqvcdjk7`
+
+#### Constitutional Committee Cold Credential
+
+Key - `00000000000000000000000000000000000000000000000000000000`
+Type - `Script Hash`
+
+Identifier - `1300000000000000000000000000000000000000000000000000000000`
+
+Bech32 - `cc_cold1zvqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6kflvs`
+
+#### DRep Credential
+
+Key - `00000000000000000000000000000000000000000000000000000000`
+Type - `Key Hash`
+
+Identifier - `2200000000000000000000000000000000000000000000000000000000`
+
+Bech32 - `drep1ygqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq7vlc9n`
+
+## Rationale: how does this CIP achieve its goals?
+
+By introducing a header and payload design for different keys, we can achieve specifying credentials in a hex format (bytes), being able to share and use across tooling is possible. We also specify bech32 prefixes for user friendly identification of the identifiers.
+
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] implemented by various tools
```

Comment:
```suggestion
- [ ] Implemented by operator tooling, including:
  - [ ] StakePool Operator Scripts
  - [ ] CNTools
- [ ] Implemented in development libraries, including:
  - [ ] Koios API
```

Reply:
this is added in implementation plan

Reply:
https://github.com/cardano-foundation/CIPs/pull/857/commits/0f4cd78a424e19d7831225b79b53d7ca67ebc31d

Reply:
@ashisherc the CIP process has a routine uncertainty here between Acceptance Criteria (criteria whose completion shows a CIP is `Active`) and Implementation Plan (the prerequisites & goals from CIP creation until declaring it `Active`).  We've tried to explain the difference here: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#path-to-active

I made my last comment without realising that you had put your entire acceptance criteria (criteria list) as an implementation plan.  So you will have to
1. move that checklist up to Acceptance Criteria (where you will see it is in other CIPs);
2. produce a proper Implementation Plan.

Since # 2 depends on how you & your associates intend to work on the CIP, I think we should talk about the Implementation Plan at the next CIP meeting if you don't yet have any ideas to post here.  I've put it up for Review at the next meeting (https://hackmd.io/@cip-editors/94); if you can't attend it then I'll bring up the issue myself and note any suggestions back here.  cc @Ryun1